src/editor/C4ConsoleWin32.cpp

```diff
@@ -1139,12 +1139,12 @@ class C4ConsoleGUIPreviewWindow : public C4Window
 		pSurface = new C4Surface(&Application, this);
 	}
 
-	~C4ConsoleGUIPreviewWindow()
+	~C4ConsoleGUIPreviewWindow() override
```

Comment:
Ultimately I think the LLVM devs have a more compelling argument, so I think we should keep the `override`.